src/Form/TypeExtension/FormTypeExtension.php

```diff
@@ -30,9 +31,11 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'cast_null_to_string' => false,
                 'form_full' => false,
+                'error_raw' => false,
```

Comment:
I do think it's really and has nothing to do in form types. This is purely twig logic. IMO it should go directly in twig template like this for example :

```twig
{{ form_errors(form, {output_raw: true}) }}
```

Perhaps there is an even better solution. What do you think @illusionOfParadise @Remg @ottaviano @thomasbisignani?

Reply:
I like this solution, keep this logic only in templates templates

Reply:
In same logic, all translation_domain, attr, label, placeholder, disable_fields, with_character_count, choice_attr, ... have to be removed ?

Reply:
It is recommended to manage them in twig directly, like button etc. You can check Symfony best practices.

Reply:
I moved set value in twig, but we need the declaration in formtypeextension

Reply:
No we don't, just test if the variable is set or not. https://twig.symfony.com/doc/2.x/tests/defined.html

Reply:
then we should have the same thing for attr label ... then ? Why it's not like that ?

Reply:
I guess it's because it is considered a real option even if it's entirely related to the view. It should be ok when you want to develop in RAD mode.

Anyway the framework is trying to solve the problem of 80% of ppl. Here we just want to solve our problem :D

Reply:
A form type is also responsible for configuring the form view, I d'ont see no harm to let an option set it, especially when it helps keeping templating simple.

Reply:
Do as you wish @cacahouwete :+1: not a big deal.